docker/base-deps/Dockerfile

```diff
@@ -0,0 +1,16 @@
+# The deploy Docker image build a self-contained Ray instance suitable
+# for end users.
+
+FROM ubuntu:xenial
+RUN apt-get update
+RUN apt-get install -y vim git wget
+RUN apt-get install -y cmake build-essential autoconf curl libtool libboost-all-dev unzip
```

Comment:
Ok, I'm updating this. It looks like the temporary files removed are pretty small in this case (already dealing with Anaconda on one line) but I'd like to stick with the pattern.